SillyBusinessInc/SillyBusinessGame
# Feat:#117 random upgrade UI

## Purpose of this PR:
This pr is made so that you can have a interactable, that opens the ui for the upgrades, and then you can add upgrades to the list (right now it has to be 3), and then the upgrades will be shown and the funntionality will run of the upgrades
## How to test:
Make a cube add InteractableUpgradeUI, then add 3 upgrades to the list, and then you should have a pop up off the upgrades, then press 1,2,3 and then you should have the upgrade depending on the input (every upgrade must have 1 action atleast)

### links: 
https://github.com/SillyBusinessInc/SillyBusinessGame/issues/117
